### PR TITLE
Introduced new widgets in header and footer

### PR DESCRIPTION
**Pull request by rushijagani**
New widgets are introduced for Header and Footer layout.

### CODE REVIEW

**Comment by Swapnildhanrale** [comment]
@maheshwaghmare I have just check this all are working branch is ready to merge.

File: inc/extras.php [hunk 1]
```diff
@@ -252,6 +256,39 @@ function ast_get_custom_html( $option_name = '' ) {
...
+		if ( 'header-main-rt-section' == $option_name ) {
+			$widget_id = 'header-widget';
+		}
+		if ( 'footer-sml-section-1' == $option_name ) {
+			$widget_id = 'footer-widget-1';
+		} elseif ( 'footer-sml-section-2' == $option_name ) {
+			$widget_id = 'footer-widget-2';
+		}
+
+		echo '<div class="ast-' . $widget_id . '-area">';
```
Comment (maheshwaghmare) [inline inc/extras.php]: @rushijagani Use `esc_attr( $widget_id )` instead of `$widget_id`.

File: inc/extras.php [hunk 2]
```diff
@@ -819,3 +859,31 @@ function ast_the_excerpt() {
...
+	 */
+	function ast_get_sidebar( $sidebar_id ) {
+		if ( is_active_sidebar( $sidebar_id ) ) {
+			dynamic_sidebar( $sidebar_id );
+		} elseif ( current_user_can( 'edit_theme_options' ) ) { ?>
+			<div class="widget ast-no-widget-row">
+				<h2 class='widget-title'><?php echo str_replace( '-', ' ', $sidebar_id ); ?></h2>
+				<p class='no-widget-text'>
+					<a href='<?php echo esc_url( admin_url( 'widgets.php' ) ); ?>'>
+						<?php echo __( 'Click here to assign a widget for this area.', 'astra' ); ?>
```
Comment (maheshwaghmare) [inline inc/extras.php]: @rushijagani Use `_e` instead of `echo __`.

File: sass/site/header/_header-common.scss [hunk 3]
```diff
@@ -67,3 +67,31 @@
 .site-logo-img img {
     @include ast-transition();
 }
+
+/**
+ * 7 - Header Widget Area
+ */
+ ul.submenu-with-border{
```
Comment (maheshwaghmare) [inline sass/site/header/_header-common.scss]: @rushijagani Use `.submenu-with-border` instead of `ul.submenu-with-border`.

**Pull request merged**